js/react/showcase/App.tsx

```diff
@@ -19,6 +19,7 @@ import {
 } from "@rustlanges/react";
 import { ShowComponent } from "./ShowComponent";
 import { Fragment, useState } from "react";
+import { ProgressBar } from "@/components/progress-bar/progress-bar.component";
```

Comment:
The component should come from `@rustlanges/react`

Reply:
Good catch, I didn’t notice